meson_options.txt

```diff
@@ -37,3 +37,9 @@ option('test',
    type: 'boolean',
    value: false,
    description: 'Enable building unit tests')
+
+option('log',
+   type: 'array',
+   choices: ['', 'info', 'debug', 'warrning', 'error'],
```

Comment:
1. we can go simply true/false,  not detailed option.

Reply:
Changed boolean type